testsuite/features/step_definitions/common_steps.rb

```diff
@@ -1187,15 +1187,17 @@ def server_secret
 
 # content lifecycle steps
 When(/^I click the environment build button$/) do
-  raise "Click on environment build failed" unless find(:xpath, '//*[@id="cm-build-modal-save-button"]').click
+  raise 'Click on environment build failed' unless find(:xpath, '//*[@id="cm-build-modal-save-button"]').click
```

Comment:
Just to easy maintain those steps, I would use the same method to click in all of them, so if find_button works, let's use it in the three methods.